tests/puppeteer/pages/BO/orders/deliverySlips/index.js

```diff
@@ -0,0 +1,34 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class DeliverySlips extends BOBasePage {
+  constructor(page) {
+    super(page);
+
+    this.pageTitle = 'Delivery Slips';
+    this.errorMessageWhenGenerateFileByDate = 'No delivery slip was found for this period.';
+
+    // Delivery slips page
+    // By date form
+    this.generateByDateForm = '[name=\'slip_pdf_form\']';
+    this.dateFromInput = '#slip_pdf_form_pdf_date_from';
+    this.dateToInput = '#slip_pdf_form_pdf_date_to';
+    this.generatePdfByDateButton = `${this.generateByDateForm} .btn.btn-primary`;
+  }
+
+  /*
+  Methods
+   */
+
+  /**
+   * Generate PDF by date
+   * @param dateFrom
+   * @param dateTo
+   * @return {Promise<void>}
+   */
+  async generatePDFByDate(dateFrom = '', dateTo = '') {
+    if (dateFrom) await this.setValue(this.dateFromInput, dateFrom);
```

Comment:
JavaScript allows the omission of curly braces when a block contains only one statement. However, it is considered by many to be best practice to never omit curly braces around blocks, even when they are optional, because it can lead to bugs and reduces code clarity

I'm from these guys who consider this being a bad practice
```suggestion
    if (dateFrom) {
      await this.setValue(this.dateFromInput, dateFrom);
    }
```